#No inputs, just a ping
---
#No response